documentation/docs/developer_guide/contribution/develop_your_idea.md

```diff
@@ -0,0 +1,119 @@
+# Develop your idea
+
+After planning for the work, we enter the stage where it is time to implement the idea in the source code. 
+
+This is a diagram view of how this stage proceed:
+
+```mermaid
+   flowchart LR
+
+      %% Define all the nodes first
+      clone[Clone repository]:::uniq ;
+      locbranch(["Create local branch: <br> git checkout &lt;branchname&gt;"]):::term;
+      edits[Modify the source code]:::term;
+      add["Add your edits to git's index: <br> git add ."]:::term;
+      commit["Commit your edits locally: <br> git commit"]:::term;
+      push[Push your edits to GitHub: <br> git push]:::term;
+      pr([Create a pull request]):::github;
+      tests[\Do the tests pass?/]:::question;
+      review[Ready for review];
+
+      clone --> locbranch--> edits;
+      edits --> add --> commit --> push;
+      push --> |first time|pr --> tests;
+      push ----> |subsequent|tests;
+      push --> |iterate|edits;
+      tests -->|Yes|review;
+      tests -->|No|edits;
+
+      classDef default fill:#FFFDE7, stroke:#FFF59D;
+      classDef uniq fill:#D81B60,stroke:#880E4F,color:#FFFFFF;
+      classDef github fill:#388E3C,stroke:#1B5E20,color:#FFFFFF;
+      classDef term fill:#F44336,stroke:#B71C1C,color:#FFFFFF;
+      classDef question fill:#6D4C41,stroke:#3E2723,color:#FFFFFF; 
+
+      linkStyle 10 stroke:red,color:red;
+      linkStyle 9 stroke:green,color:darkgreen;
+
+      click review "https://cable.readthedocs.io/en/latest/developer_guide/contribution_flowchart/#review" "Review process"
+
+;
+```
```

Comment:
Can we make the font larger in the flowchart? It might also be easier if the flowchart follows from top to bottom rather than left to right.

Reply:
That's funny. It was initially top to bottom and I changed it to left to right to gain space. But I'll put it back top to bottom. I don't think I can change the